vector/src/main/res/values/strings.xml

```diff
@@ -1051,6 +1051,10 @@
     <string name="tchap_burger_menu_contacts">Contacts</string>
     <string name="tchap_burger_menu_public_rooms">Public rooms</string>
     <string name="tchap_burger_menu_info">This app is not approved for\nrestricted information interchange</string>
+    <string name="tchap_error_message_default">An error occurred, please try later</string>
+
+    // tchap discussion
+    <string name="tchap_cannot_invite_deactivated_account_user">The remote user left definitely this room</string>
```

Comment:
It's a bit more than that (account deactiviated) but I assume this wording is ok for you

Reply:
Yes, I did not mention "deactivated account" on purpose, my plan is just to inform that talking in this room is useless without providing the actual reason which may be obscured for the end user ;)
I will sync about with DINSIC later